packages/server/src/api/index.ts

```diff
@@ -576,14 +576,13 @@ router.put('/user/article_series/:id', async (req, res) => {
  * Delete article series
  */
 router.delete('/user/article_series/:id', async (req, res) => {
-    if (!('id' in req.params)) {
-        res.status(400).send({ error: 'Id is required' });
-        return;
-    }
 
-    if (!req.params.id) {
-        // id cannot be 0
-        res.status(400).send({ error: 'Id is required' });
+    const idFromParams = z.coerce.number().int().positive();
```

Comment:
There is already `id zod validator` try to use that and all the zod-related validation logic should reside in `validators` folder.